src/AssignSubstitutes/ThePointlessClassIMade.java

```diff
@@ -289,16 +289,16 @@ private void StringFormatReplacement(String format, String teacherName){
         notAbsent.add(0, nullTeacher);
         for(Teacher t: notAbsent) {
             System.out.print(t + " ");
-            /*for (Period p : t.getSchedule()) {
+            for (Period p : t.getSchedule()) {
                 System.out.print(p.getPeriodNumber() + " ");
-            }*/
+            }
             System.out.println();
         }
         return notAbsent;
-    }
+    }*/
     //For test use until InformationHandle is available
     //gets a list of available teachers for a given period
-    public static ObservableList<ArrayList<Object>> getAvailabilityStats(Collection<OnStaffTeacher> fullTeacherList) throws Exception{
+    /*public static ObservableList<ArrayList<Object>> getAvailabilityStats(Collection<OnStaffTeacher> fullTeacherList) throws Exception{
```

Comment:
Code will not compile with getAvailabilityByDate commented out. Why comment any of this code out at all. It now exists elsewhere, but it should be plain to see in an IDE, and some editors which methods are being referenced. Large commented out code blocks like this are hard to manage.